internal/pkg/cli/pipeline_status.go

```diff
@@ -144,22 +145,6 @@ func (o *pipelineStatusOpts) askAppName() error {
 }
 
 func (o *pipelineStatusOpts) askPipelineName() error {
```

Comment:
Not to address here but perhaps latter some day we could consider moving the parameter from the receiver to function arg if all of these `pipeline` commands have the same `askPipelineName`. E.g. [here](https://github.com/aws/copilot-cli/blob/76e100c2fb7955125cada30bfcd5d9bd8ed269e6/internal/pkg/cli/svc_deploy.go#L338)